sql/src/test/java/io/crate/integrationtests/KillIntegrationTest.java

```diff
@@ -123,4 +135,32 @@ public void testKillGroupBy() throws Exception {
         runJobContextReapers();
     }
 
+    @Test
+    public void testKillCopyToJob() throws Exception {
+        String path = temporaryFolder.newFolder().getAbsolutePath();
+        setup.setUpEmployees();
+        assertGotCancelled("copy employees to directory ?", new Object[]{path});
+    }
+
+    @Test
+    public void testKillInsetJob() throws Exception {
+        setup.setUpEmployees();
+        execute("create table new_employees (" +
+                " name string, " +
+                " department string," +
+                " hired timestamp, " +
+                " age short," +
+                " income double, " +
+                " good boolean" +
+                ") with (number_of_replicas=1)");
+        ensureYellow();
+        assertGotCancelled("insert into new_employees (select * from employees)", null);
```

Comment:
imho it would be better to have a new assertion that just uses the new kill 'jobId', otherwise the behaviour of existing tests is also changed.